code/game/objects/items/toys.dm

```diff
@@ -1593,75 +1613,166 @@
 		input = GRAB
 	if(icon_x < ICON_SPLIT && icon_y < ICON_SPLIT)
 		input = HARM
-	render(input)
-	player_sequence += input
+
+	player_input(user, input)
+
+/obj/item/toy/intento/proc/boot()
+	say("Game starting!")
+	playsound(src, 'sound/machines/synth_yes.ogg', 50, FALSE)
+
+	state = STATE_STARTING
+	next_process = world.time + TIME_TO_BEGIN
+	START_PROCESSING(SSfastprocess, src)
+
+/obj/item/toy/intento/proc/player_input(mob/player, intent)
+	// All branches of this proc lead to us wanting to process
+	START_PROCESSING(SSfastprocess, src)
+
+	render(intent)
+
+	player_sequence += intent
 	for(var/i in 1 to player_sequence.len)
 		if(player_sequence[i] != current_sequence[i])
-			return end(user)
+			state = STATE_END_OF_GAME
+			next_process = world.time + TIME_TO_RESET_ICON
+			return
+
 	if(player_sequence.len == current_sequence.len)
 		score++
-		addtimer(CALLBACK(src, .proc/start_round, 1 SECONDS))
+
+		state = STATE_STARTING
+		next_process = world.time + TIME_TO_BEGIN
```

Comment:
Use the cooldown defines.

Reply:
Or use `next_process = TIME_TO_BEGIN` then `next_process -= delta_time`, and check if `next_process <= 0`.